diagnostic.md

```diff
@@ -94,4 +94,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+'exercise'
+'Jason' 'Weeks'
```

Comment:
Very close. teamMember['name'] would actually return the entire 'name' object, which would be {given: 'Jason', surname: 'Weeks'}